dune-project

```diff
@@ -27,7 +28,9 @@
   (domain_shims (>= 0.1.0))
   (alcotest (and (>= 1.7.0) :with-test))
   (qcheck (and (>= 0.18.1) :with-test))
+  (backoff (>= 0.1.0))
```

Comment:
If you run `dune build` (for example) then `dune` will regenerate the `saturn_lockfree.opam` file and this dependency should then be included in the opam file.